src/transform/xsl/default.xsl

```diff
@@ -430,18 +430,18 @@
     </xsl:template>
 
     <xsl:template match="fn-group/fn">
-        <li>
+        <xsl:variable name="fn-number">
+            <xsl:number level="any" count="fn[not(ancestor::front)]" from="article | sub-article | response"/>
+        </xsl:variable>
+        <li id="fn{$fn-number}">
+            <span class="footnotemarker" id="n{$fn-number}"></span>
             <xsl:apply-templates/>
+            [<a href="#nm{$fn-number}"><sup>^</sup></a>]
```

Comment:
The span now closes before this is rendered, so the footnotemarker is no longer inside an element with the class `footnotemarker` which might screw up CSS overrides

Reply:
From what I can see there are no references to footnotemarker, I could add it to the <a>?

Reply:
> From what I can see there are no references to footnotemarker in the css, I could add it to the though?

I believe some of the new journals used it to tweak the colour and size so it is worth adding it to the `<a>` here.

Also, not much point having it on the `<span>>` anymore since it is empty